packages/peregrine/lib/talons/AuthBar/useAuthBar.js

```diff
@@ -0,0 +1,33 @@
+import { useCallback } from 'react';
+import { useUserContext } from '../../context/user';
+
+/**
+ * Returns props necessary to render an AuthBar component.
+ *
+ * @param {object} props
+ * @param {boolean} props.disabled - whether sign in button should be disabled
+ * @param {function} props.showMyAccount - callback that displays my account view
+ * @param {function} props.showSignIn - callback that displays sign in view
+
+ * @return {{ currentUser: object, handleClick: function, isSignedIn: boolean, isSignInDisabled: boolean, showMyAccount: function }}
```

Comment:
This comment doesn't match what's returned from the function

Reply:
Also this is a bit hard to read. Would be extra 🥇 if these were broken out and described like `props` is above.

Reply:
Oops.